### ISSUE TITLE
Next 13.1.7-canary.1 build fails on Vercel when sass is used

### ISSUE THREAD

**Issue by ervwalter** [vercel/next.js#45462]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

    Operating System:
      Platform: linux
      Arch: x64
      Version: #62-Ubuntu SMP Tue Nov 22 19:54:14 UTC 2022
    Binaries:
      Node: 16.19.0
      npm: 8.19.3
      Yarn: N/A
      pnpm: 7.26.2
    Relevant packages:
      next: 13.1.7-canary.1
      eslint-config-next: 13.1.6
      react: 18.2.0
      react-dom: 18.2.0

### Link to the code that reproduces this issue

https://github.com/trendweight/trendweight/tree/test-build

### To Reproduce

Deploy with Vercel

### Describe the Bug

Build fails on Vercel with this error:

```
info  - Creating an optimized production build...
unhandledRejection Error: Cannot find module '/vercel/path0/node_modules/.pnpm/next@13.1.7-canary.1_3duxcpitbtplz62feflag7fwby/node_modules/next/dist/compiled/sass-loader/fibers.js'
```

Note, I also tried using yarn instead of pnpm and a similar error occurs, so this does not appear to be pnpm specific.

Error does not occur when building locally.

Build log:  https://vercel.com/twistymaze/trendweight/rrxmV3viSkV6thJCyGSP2EJpJPhx

### Expected Behavior

Build succeeds

### How are you deploying your application? (if relevant)

Vercel

**Comment by Trallp** [comment]
Hi! Try node >=16 
related https://github.com/vercel/next.js/issues/45052

**Comment by ervwalter** [comment]
Thanks.  I was able to change Vercel build settings to use Node 18 and it started building successfully.

**Comment by jiggy1com** [comment]
@ervwalter - same! I was using node 14.x.x and sass failed. `nvm use 18 `=> success

**Comment by shuding** [comment]
We can close this as it should be addressed by #45052 (upgrading Node works too).
